xep-0388.xml

```diff
@@ -15,11 +15,31 @@
   <sig>Standards</sig>
   <dependencies>
     <spec>XMPP Core</spec>
+    <spec>rfc5802</spec>
+    <spec>XEP-0440</spec>
   </dependencies>
   <supersedes/>
   <supersededby/>
   <shortname>sasl2</shortname>
   &dcridland;
+  &tmolitor;
+  &mwild;
+  <revision>
+    <version>0.4.0</version>
+    <date>2022-10-06</date>
+    <initials>tm</initials>
+    <remark>
+      <ul>
+        <li>Bump namespace</li>
+        <li>Make SCRAM hashes (and other SASL mechanism families) upgradable</li>
```

Comment:
Well, yes and no. The upgrade tasks themselves probably still belong into SASL2. SCRAM updates on the other hand could be in an extra XEP as well.

I merely added them to SASL2 because it was easier and the SCRAM update part was relatively small compared to the rest of the SASL2 XEP.

Reply:
I've split out the SCRAM upgrade task specification into a new ProtoXEP.

Reply:
@horazont over here